lib/util/deep-assign.js

```diff
@@ -23,6 +23,6 @@ export default function deepAssign (target, ...sources) {
   return deepAssign(target, ...sources);
 }
 
-function isObject (item) {
-  return item && typeof item === 'object' && !Array.isArray(item);
+function isAssignableObject (item) {
+  return item && typeof item === 'object' && !Array.isArray(item) && !(item instanceof Node);
```

Comment:
I think checking for `item` and `typeof item === 'object'` is redundant?

Reply:
The object check is to ensure it isn't something like a `Boolean` or `String`, since we don't want to assign on those.

Reply:
hmm yes, the first op can be removed 👍